src/gate_detect/README.md

```diff
@@ -0,0 +1,88 @@
+##Purpose
```

Comment:
Needs to be `## Purpose` for markdown (there are several instances of this, just pointing out this one).

Reply:
done